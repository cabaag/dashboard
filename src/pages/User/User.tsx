import React from 'react';

// tslint:disable-next-line: variable-name
const User = () => {
	return (
		<div>User</div>
	);
};

export default User;
