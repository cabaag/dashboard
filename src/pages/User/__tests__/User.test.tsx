import { shallow } from 'enzyme';
import React from 'react';
import User from '../User';

describe('test User component', () => {

		test('should render component', () => {
				expect(shallow(<User />)).toMatchSnapshot();
		});

});
