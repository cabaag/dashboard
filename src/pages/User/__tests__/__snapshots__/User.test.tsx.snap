// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test User component should render component 1`] = `ShallowWrapper {}`;
